File: src/main/java/com/udacity/store/model/Order.java
package com.udacity.store.model;

import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.*;

@Entity
@Table(name = "orders")
@Data
@NoArgsConstructor
public class Order {
    //TODO: Add the details of the order class

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    long id;

    @Column(name = "name")
    String name;

    @Column(name = "total")
    double total;

    @Column(name = "address")
    String address;

    @Column(name = "creditCart")
    String creditCart;


    public Order(String name, double total, String address, String creditCart) {
        this.name = name;
        this.total = total;
        this.address = address;
        this.creditCart = creditCart;
    }
}

File: src/main/webapp/angular-frontend/src/app/models/Order.ts
//TODO: Add the contents of the order class
export class Order {
  id: number;
  name: string;
  address: string;
  creditCart: string;
  total: number;

  constructor() {
    this.id = 0;
    this.name = '';
    this.address = '';
    this.creditCart = '';
    this.total = 0;
  }
}
